conversions/weight_conversion.py

```diff
@@ -0,0 +1,64 @@
+"""
+Functions useful for conversion fo weight units
```

Comment:
```suggestion
Conversion of weight units.
```
Please provide the usage of the function, like how would an unknown person use this.